Compilation for ARM fails in DEBUG mode.

https://builds.cuberite.org/job/cuberite/job/master/job/linux-odroid-c1/job/debug/6/console

```
/home/jenkins/workspace/cuberite/master/odroid-c1/debug/lib/polarssl/library/bignum.c: In function ‘mpi_mul_hlp’:
/home/jenkins/workspace/cuberite/master/odroid-c1/debug/lib/polarssl/library/bignum.c:1131:1: error: r7 cannot be used in asm here
 }
 ^
lib/polarssl/library/CMakeFiles/mbedtls.dir/build.make:192: recipe for target 'lib/polarssl/library/CMakeFiles/mbedtls.dir/bignum.c.o' failed
```

--- a/SetFlags.cmake
+++ b/SetFlags.cmake
@@ -108,6 +108,8 @@ macro(set_flags)
 		endif()
 
 		if("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "arm")
+			# mbed TLS uses the frame pointer's register in inline assembly:
+			# https://tls.mbed.org/kb/development/arm-thumb-error-r7-cannot-be-used-in-asm-here
 			set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -fomit-frame-pointer")
 			set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -fomit-frame-pointer")
 		endif()

--- a/SetFlags.cmake
+++ b/SetFlags.cmake
@@ -109,6 +109,7 @@ macro(set_flags)
 
 		if("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "arm")
 			set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -fomit-frame-pointer")
+			set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -fomit-frame-pointer")
 		endif()
 
 		set(CMAKE_CXX_FLAGS          "${CMAKE_CXX_FLAGS}          -std=c++11")

